casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala

```diff
@@ -90,9 +90,16 @@ class MultiParentCasperAddBlockSpec extends AnyFlatSpec with Matchers with Inspe
                     shardId = SHARD_ID
                   )
         signedBlock2 <- node.addBlock(deploy2)
+        blockData    = BlockData.fromBlock(signedBlock2)
+        rand         = BlockRandomSeed.fromBlock(signedBlock2)
+
         data <- getDataAtPrivateChannel[Effect](
                  signedBlock2,
-                 Base16.encode(Tools.unforgeableNameRng(deploy2.pk, deploy2.data.timestamp).next())
+                 rand
+                   .splitByte(0.toByte)
+                   .splitByte(BlockRandomSeed.UserDeploySplitIndex)
```

Comment:
We need good comments for every split of random generator, otherwise it will be difficult for understand the code later or for new developers.

Reply:
I use some named values. Do you think that's enough?